scripts/system_info.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/python
```

Comment:
I believe I asked you about this but never followed through.   It belongs in magni robot but knowing that the merge there is so ancient I was concerned it may not get there ... ever.    I am happy to move it BUT want BADLY that this tool, crude as it is, be out in field on a very recent image.  We can discuss and I'll take it out of this pull request